components/Mailer/basic_usage.rst

```diff
@@ -0,0 +1,193 @@
+Basic usage
+===========
+
+.. _component_mailer_provider_default-settings-provider:
+
+DefaultSettingsProvider
+-----------------------
+
+The **DefaultSettingsProvider** is simple model of default settings for **Sender**.
+
+.. code:: php
+
+    <?php
+
+    use Sylius\Component\Mailer\Provider\DefaultSettingsProvider;
+
+    $defaultSettingsProvider = new DefaultSettingsProvider('Mike Ehrmantraut', 'mike.ehrmantraut@gmail.com');
+
+    $defaultSettingsProvider->getSenderAddress(); // mike.ehrmantraut@gmail.com
+    $defaultSettingsProvider->getSenderName(); // Output will be Mike Ehrmantraut
+
+.. note::
+
+.. _component_mailer_provider_email-provider:
+
+EmailProvider
+-------------
+
+The **EmailProvider** allows you to get specific email from storage.
+
+.. code:: php
+
+    <?php
+
+    use Sylius\Component\Mailer\Provider\EmailProvider;
+    use Sylius\Component\Resource\Repository\InMemoryRepository;
+
+    $inMemoryRepository = new InMemoryRepository();
+
+    $configuration = array(
+        '12' => array(
+            'subject' => 'Christmas party',
+            'template' => 'email.html.twig',
+            'enabled' => true,
+            'sender' => array(
+                'name' => 'John',
+                'address' => 'john.doe@gmail.com',
+            ),
+        ),
+    );
+
+    $emailProvider = new EmailProvider($inMemoryRepository, $configuration);
+
+    $email = $emailProvider->getEmail('12'); // This method will search for an email in your storage or in given configuration.
+
+    $email->getCode(); // Output will be 12.
+    $email->getSenderAddress(); // Output will be john.doe@gmail.com.
+    $email->getSenderName(); // Output will be John.
+
+Renderer
+--------
+
+.. _component_mailer_renderer_rendered-email:
+
+RenderedEmail
+~~~~~~~~~~~~~
```

Comment:
I'd move that somewhere further, it is super simple and can be treated more as an extra info for the renderer, than a separate model chapter. Agree?